apps/api/src/app/benchmark/benchmark.controller.ts

```diff
@@ -94,4 +94,41 @@ export class BenchmarkController {
       );
     }
   }
+
+  @Post('remove')
```

Comment:
Can you change this to `@Delete(':dataSource/:symbol')`?

Reply:
Resolved in latest commit.